#!/usr/bin/env python
PACKAGE = "my_car"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# name, type, level, "description", default, min, max

gen.add("x_init",    double_t,    0, "Parameter for the x coord", 0,  0, 100)  #set to 0 to default
gen.add("y_init",    double_t,    1, "Parameter for the y coord", 0,  0, 100)  #set to 0 to default
gen.add("theta",    double_t,    2, "Parameter for theta", 0,  0, 360)  #set to 0 to default
gen.add("odometry",    int_t,    3, "Parameter for the odomemtry", 1,  0, 1)  #set to 0 to default
 

exit(gen.generate(PACKAGE, "car_listener", "parameters"))
